build.assets/Dockerfile-centos7-fips

```diff
@@ -91,6 +127,17 @@ RUN cd / && curl -L https://s3.amazonaws.com/clientbuilds.gravitational.io/go/ce
 # Copy libbpf into the final image.
 COPY --from=libbpf /opt/libbpf/usr /usr
 
+# Install Rust using the ci user, as that is the user that
+# will run builds using the Rust toolchains we install here.
+# Cross-compilation targets are only installed on amd64, as
+# this image doesn't contain gcc-multilib.
 USER ci
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal --default-toolchain $RUST_VERSION && \
+    rustup --version && \
+    cargo --version && \
+    rustc --version && \
+    rustup component add rustfmt clippy && \
+    if [ "$BUILDARCH" = "amd64" ]; then rustup target add aarch64-unknown-linux-gnu; fi
```

Comment:
This looks like a bug. `aarch64`- is ARM, but you're checking `amd64` buildarch.

Reply:
@zmb3 This is correct. `BUILDARCH` is the arch of the Docker image. We always build for amd64 on Intel. The only exception is Mac, but we don't use that image on Mac AFAIK. 
@LKozlowski  Can we skip that check at all and just install the toolchain? I don't think this few hundred megs is so painful in fips image.

Reply:
Sure! To be honest I copied that whole rust section from this Dockerfile and assumed it is correct: https://github.com/gravitational/teleport/blob/master/build.assets/Dockerfile#L252

Does it mean that we also have a bug here?

Reply:
@LKozlowski This condition makes sense in the main Dockerfile. We cross-compile arm64 only on amd64, so arm64 toolchain is not required for anything other than amd64.
BTW. arm64 and amd64 are very confusing names...